### PR TITLE
Omniauth gitlab integration

### PR DESCRIPTION
**Pull request by sreecodeslayer**
* User can login with gitlab oauth
* Saves basic user info and the access token to list user's repos later on

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -62,8 +62,6 @@ GEM
     bootsnap (1.4.5)
       msgpack (~> 1.0)
     builder (3.2.4)
-    bulma-rails (0.8.0)
```
Comment (thegeorgeous) [inline Gemfile.lock]: Is this gem being used?

File: app/assets/stylesheets/me.scss [hunk 2]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the Me controller here.
```
Comment (thegeorgeous) [inline app/assets/stylesheets/me.scss]: This file seems unnecessary

File: .envrc [hunk 3]
```diff
@@ -0,0 +1,3 @@
+export GITLAB_KEY=your-key
```
Comment (thegeorgeous) [inline .envrc]: It might be easier to do this with the `dotenv` gem

Comment (sreecodeslayer) [reply .envrc]: yea, I just pushed this since I use `direnv` everywhere 🙂

File: app/assets/stylesheets/me.scss (hunk 2 above)
Comment (sreecodeslayer) [reply app/assets/stylesheets/me.scss]: Oops, will remove. was from the controller generator

File: Gemfile.lock (hunk 1 above)
Comment (sreecodeslayer) [reply Gemfile.lock]: no, i later changed from this to bulma cdn based theme. viz why i removed it

File: app/assets/stylesheets/me.scss (hunk 2 above)
Comment (sreecodeslayer) [reply app/assets/stylesheets/me.scss]: remove them!

**Pull request merged**